google/cloud/forseti/services/cli.py

```diff
@@ -23,14 +23,20 @@
 from google.protobuf.json_format import MessageToJson
 
 from google.cloud.forseti.services import client as iam_client
+from google.cloud.forseti.common.util import log_util
 
+LOGGER = log_util.get_logger(__name__)
 
 def define_playground_parser(parent):
     """Define the playground service parser.
 
     Args:
         parent (argparser): Parent parser to hook into.
     """
+
+    if parent is None:
+        LOGGER.warn("parent is None")
```

Comment:
In general, I'm not sure it is worthwhile to log a missing parameter if the function or method will throw an AttributeError exception due to the missing parameter. It may be better to use assert(parent is not None,  'parent is None') so these can be caught during testing, but are not checked when python is started with -O if we want to run without assertions.

This applies to all the locations below where the log message is just noting missing parameters.

Reply:
I will go ahead and remove the unnecessary log statements but I don't think it's a good idea to put  assert statements in between the code, do you mind if i just leave that part out?

Reply:
That's fine, we don't make a lot of use of assert today, so no reason to add it here now.